lms/djangoapps/student_account/views.py

```diff
@@ -302,6 +306,69 @@ def _external_auth_intercept(request, mode):
         return external_auth_register(request)
 
 
+def get_user_orders(user):
```

Comment:
@ahsan-ul-haq a lot of this duplicates code from https://github.com/edx/edx-platform/blob/master/openedx/core/lib/edx_api_utils.py#L14. Have you check to see if there are parts you can re-use?